src/main/scala/com/gu/flexible/snapshotter/config/SchedulerConfig.scala

```diff
@@ -16,13 +16,13 @@ case class SchedulerConfig(
   stack: String) extends CommonConfig
 
 object SchedulerConfig extends Logging {
-  def resolve(stage: String, context: Context)(implicit lambdaClient: AWSLambda): SchedulerConfig = {
-    val lambdaJson = LambdaConfig.getDescriptionJson(context)
-    val lambdaConfig = lambdaJson.as[LambdaSchedulerConfig]
-    SchedulerConfig(
-      snsTopicArn = lambdaConfig.snsTopicArn,
-      stage = stage,
-      stack = lambdaConfig.stack
-    )
-  }
-}
\ No newline at end of file
+  def resolve(): Option[SchedulerConfig] = for {
```

Comment:
Be tempted to write something here that tells us what's missing – we know we're going to fail fast downstream, and at the moment IIUC we'll blow up with a nondescript `java.util.NoSuchElementException`. It'd be nice to know , via logging or the exception itself, what's missing.